### PR TITLE
Update documentation on min Java Version Required #1448

### LINKED ISSUE #1448: Update documentation on min Java Version Required
Also, put a comment near the variable of java version required to update the docs.

### PR DESCRIPTION
**Pull request by HansNewbie**
Fixes #1448

### CODE REVIEW

**Comment by HansNewbie** [comment]
@dariusf ready for review

**Comment by dariusf** [comment]
Looks good. Would it be a good idea to document why we need this particular version somewhere?

**Comment by HansNewbie** [comment]
Sure, but I am not sure which doc file to put in. [toolStack.md under core](https://github.com/HubTurbo/HubTurbo/blob/master/docs/toolStack.md#core)?

**Comment by dariusf** [comment]
Okay, sure.

**Comment by HansNewbie** [comment]
@dariusf ready for review.

File: src/main/java/ui/UI.java [hunk 1]
```diff
@@ -61,7 +61,10 @@
     private static final int REFRESH_PERIOD = 60;
 
     /**
-     * Minimum Java Version Required by HT
+     * Minimum Java Version Required by HT.
+     *
+     * If this is changed, update gettingStarted.md on Java version required and
+     * toolStack.md on reason for minimum Java version.
```
Comment (damithc) [inline src/main/java/ui/UI.java]: This comment can be missed. May be add as a `//comment` to the end of line 69?

File: docs/toolStack.md [hunk 2]
```diff
@@ -11,6 +11,8 @@ HubTurbo is written in Java. [ControlsFX](http://fxexperience.com/controlsfx/) e
 
 Platform-specific components are used as well: [Selenium](http://www.seleniumhq.org/) for instantiating and controlling a browser window, [JNA](https://github.com/twall/jna) to manage window focus on Windows, and [jkeymaster](https://github.com/tulskiy/jkeymaster) for a global hotkey.
 
+Due to usage of ControlsFX 8.40.10, HubTurbo requires minimum Java version of 1.8.0_60.
```
Comment (damithc) [inline docs/toolStack.md]: Here we are creating another place where things can become inconsistent. Perhaps this too can be added as a comment where the min version is declared?

**Comment by damithc** [comment]
Some cosmetic changes requested.

File: docs/toolStack.md (hunk 2 above)
Comment (HansNewbie) [inline docs/toolStack.md]: Sure.

**Pull request merged**